### DISCUSSION TITLE
Inform dark-reader that the website already supports dark theme builtin

### DISCUSSION THREAD

**Discussion by avestura** [darkreader/darkreader#7321] [Ideas]
Hello

Do you guys think if it's a good idea to somehow inform darkreader that the website is already on a dark theme or at least supports dark theme built-in?
This can be done using a meta tag:

```html
<meta name="darkreader" content="supports-dark,is-dark" />
```

Darkreader might refuse to modify the theme of the page with respect to user configuration. User might want to:
1. **Prevent modification**: Respect the website's wish by stopping the darkreader from changing the theme of the page and show it in the dark theme version provided by the site itself
2. **Override theme:** Ignore the meta tag and always apply darkreader modifications

![image](https://user-images.githubusercontent.com/9091958/140994136-65c1ef41-76d5-4a82-940c-d70d1d2a1649.png)

**Comment by Gusted** [comment]
We're tracking this issue over at #1536
